python/conversion/format.py

```diff
@@ -11,13 +13,20 @@
     import conversion.prophesee
 except ImportError:
     print("Conversion from .raw is not possible. If you want to extract .raw files, please install Metavision 2.2")
-    metavision_found = False
-
+    metavision_found_2_2 = False
+try:
+    import conversion.prophesee_dat
+except ImportError:
+    print("Conversion from .dat is not possible. If you want to extract .dat files, please install Metavision 2.0")
+    metavision_found_2_0 = False
 
 def get_generator(input_file: Path, delta_t_ms: int=1000, topic: str='/dvs/events'):
     if input_file.suffix == '.raw':
-        assert metavision_found, 'Could not find Metavision packages'
+        assert metavision_found_2_2, 'Could not find Metavision 2.2 packages'
```

Comment:
Not sure if the string is correct. It's more about the fact that we could not import the module for raw conversion